.circleci/config.yml

```diff
@@ -228,28 +252,39 @@ jobs:
     steps:
       - checkout
       - run: REPO_DOMAIN=kubeapps REPO_NAME=kubeapps ./script/create_release.sh ${CIRCLE_TAG}
-  GKE_1_9_MASTER:
+  GKE_1_9:
     <<: *gke_test
     environment:
       HELM_VERSION: v2.11.0
       GKE_BRANCH: 1.9
-  GKE_1_10_MASTER:
-    <<: *gke_test
-    environment:
-      HELM_VERSION: v2.11.0
-      GKE_BRANCH: 1.10
   GKE_1_9_LATEST_RELEASE:
     <<: *gke_test
     environment:
       HELM_VERSION: v2.11.0
       GKE_BRANCH: 1.9
       # We want to also test the chart with the current release of the images
       TEST_LATEST_RELEASE: 1
+  GKE_1_10:
+    <<: *gke_test
+    environment:
+      HELM_VERSION: v2.11.0
+      GKE_BRANCH: "1.10"
   GKE_1_10_LATEST_RELEASE:
     <<: *gke_test
     environment:
       HELM_VERSION: v2.11.0
-      GKE_BRANCH: 1.10
+      GKE_BRANCH: "1.10"
```

Comment:
Can you write a comment explaining why you need quotes? 

I also think that it might make sense to quote them all?